roxie/ccd/ccdserver.cpp

```diff
@@ -2704,27 +2766,63 @@ class CRoxieServerTwoInputActivity : public CRoxieServerActivity
 
     unsigned sourceIdx1 = 0;
     Owned<CRoxieServerReadAheadInput> puller;
-
+    bool startInputsSequentially = false;
 public:
     CRoxieServerTwoInputActivity(IRoxieAgentContext *_ctx, const IRoxieServerActivityFactory *_factory, IProbeManager *_probeManager)
         : CRoxieServerActivity(_ctx, _factory, _probeManager)
     {
         input1 = NULL;
         inputStream1 = NULL;
+        startInputsSequentially = factory->startInputsSequentially() || factory->queryQueryFactory().queryOptions().startInputsSequentially;
```

Comment:
minor: similar comment to executeDependencies.  cleaner to include the second half within factory->startInputsSequentially().